src/ray/core_worker/task_interface.h

```diff
@@ -1,6 +1,7 @@
 #ifndef RAY_CORE_WORKER_TASK_INTERFACE_H
 #define RAY_CORE_WORKER_TASK_INTERFACE_H
 
+#include <list>
 #include "common.h"
```

Comment:
Regarding including "common.h" here: Includes should be fully qualified, even if the file is in the same folder, i.e. it should be `#include "ray/core_worker/common.h"`, see the google styleguide:

```
All of a project's header files should be listed as descendants of the project's source directory without use of UNIX directory shortcuts . (the current directory) or .. (the parent directory). For example, google-awesome-project/src/base/logging.h should be included as:
```

Reply:
Thanks. this is a good point which can avoid including unintended files with the same name.